plugin_interpreter/src/rethink_interface.py

```diff
@@ -186,58 +161,14 @@ def check_for_plugin(self, plugin_name):
         """
         try:
             rethinkdb.db("Plugins").table(plugin_name).run(
-                self.validate_db(self.rethink_connection)
+                connect(host=self.host, port=self.port)
             )
             return True
         except rethinkdb.ReqlOpFailedError:
             return False
 
-    @staticmethod
-    def validate_db(connection):
-        """Validate database connection
-
-        This method validates that the databases
-        and tables needed for operation are available
-        in the database (which the connection argument
-        connects to).
 
-        Arguments:
-            connection {rethinkdb.connection} -- connection object
-            to the rethink database.
-
-        Returns:
-            {rethinkdb.connection} -- connection object, returned
-            if the validation passes.
-        """
-
-        queries = [
-            rethinkdb.db_list().contains("Plugins"),
-            rethinkdb.db_list().contains("Brain"),
-            rethinkdb.db_list().contains("Audit"),
-            rethinkdb.db("Brain").table("Targets"),
-            rethinkdb.db("Brain").table("Outputs"),
-            rethinkdb.db("Brain").table("Jobs"),
-            rethinkdb.db("Audit").table("Jobs")
-        ]
-
-        i = 0
-        now = time()
-        while time() - now < 15:
-            try:
-                queries[i].run(connection)
-                i += 1
-            except rethinkdb.ReqlOpFailedError:
-                sleep(0.2)
-            except rethinkdb.ReqlDriverError as err:
-                stderr.write("".join((str(err), "\n")))
-                break
-            if i >= len(queries):
-                return connection
-
-        stderr.write("DB not available!\n")
-        exit(112)
-
-    def _update_job_status(self, job_data):
```

Comment:
suggest deprecating rather than outright removing for posterity.

_update_job_status could call update_job_status

pragma nocover it because cc is rude